### PR TITLE
Update lockfile

### PR DESCRIPTION
**Pull request by harens**
Hopefully closes https://github.com/ogham/exa/issues/853.

Update the lockfile to match `cargo.toml`.

### CODE REVIEW

**Comment by ariasuni** [comment]
Please update Rust’s version to 1.45.0 in `.travis.yml` and `README.md`, like I did in [#857](https://github.com/ogham/exa/pull/857/files). You updated the `git2` version, and now it needs at least Rust 1.45.0.

**Comment by harens** [comment]
Thank you for your response @ariasuni (and for your work on this project).

I left it as a separate commit so as not to get too much in the way of #857, and so that it's possible to only cherry pick my original commit if desired.

**Comment by ariasuni** [comment]
Don’t worry, I’ll rebase my pull request after I merge this. I’m just waiting for the CI checks to finish.

File: README.md [hunk 1]
```diff
@@ -201,8 +201,8 @@ To build without Git support, run `cargo install --no-default-features exa` is a
 <a id="development">
 <h1>Development
 
-<a href="https://blog.rust-lang.org/2020/03/12/Rust-1.42.html">
-    <img src="https://img.shields.io/badge/rustc-1.42+-lightgray.svg" alt="Rust 1.42+" />
+<a href="https://blog.rust-lang.org/2020/03/12/Rust-1.45.html">
+    <img src="https://img.shields.io/badge/rustc-1.45+-lightgray.svg" alt="Rust 1.45+" />
```
Comment (ariasuni) [inline README.md:205]:
```suggestion
<a href="https://blog.rust-lang.org/2020/08/03/Rust-1.45.2.html">
    <img src="https://img.shields.io/badge/rustc-1.45.2+-lightgray.svg" alt="Rust 1.45.2+" />
```

File: README.md [hunk 2]
```diff
@@ -211,7 +211,7 @@ To build without Git support, run `cargo install --no-default-features exa` is a
 </h1></a>
 
 exa is written in [Rust](https://www.rust-lang.org/).
-You will need rustc version 1.42.0 or higher.
+You will need rustc version 1.45.0 or higher.
```
Comment (ariasuni) [inline README.md:214]:
```suggestion
You will need rustc version 1.45.2 or higher.
```

File: .travis.yml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 language: rust
 rust:
-  - 1.42.0
+  - 1.45.0
```
Comment (ariasuni) [inline .travis.yml:3]:
```suggestion
  - 1.45.2
```

**Review by ariasuni — CHANGES_REQUESTED** [review]
Let’s depend on Rust 1.45.2 since it the latest 1.45.x and it fixes bugs. Also, the link to the Rust blog was incorrect.

**Comment by ariasuni** [comment]
Aaaand it’s done! \o/ Also:

> Thank you for your response @ariasuni (and for your work on this project).

I’m glad my work is useful and appreciated :)

**Comment by harens** [comment]
Thank you for merging! 🎉 (and for correcting my mistakes...)

**Pull request merged**